modules/microshift-firewall-known-issue.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-known-issue_{context}"]
+== Known firewall issue
+To avoid breaking traffic flows with a firewall reload or restart, execute firewall commands before starting {product-title}. The CNI driver in {product-title} makes use of iptable rules for some traffic flows, such as those using the NodePort service. The iptable rules are generated and inserted by the CNI driver, but are deleted when the firewall reloads or restarts. The absence of the iptable rules breaks traffic flows. If firewall commands have to be executed after {product-title} is running, manually restart ovnkube-master pod in openshift-ovn-kubernetes namespace to reset the rules controlled by the CNI driver.
```

Comment:
Generally I notice that when we have a known issues section, we have the issue itself bulleted. So...

* To avoid breaking. . .